lib/core/sync_job_runner.rb

```diff
@@ -56,9 +56,15 @@ def cron_parser(cronline)
 
     def should_sync?
       # sync_now should have priority over cron
-      return true if @connector_settings[:sync_now] == true
+      if @connector_settings[:sync_now] == true
```

Comment:
I think we might want to clarify it, then. I assumed that even with the scheduling disabled, we might still want to sync. I mean, scheduling <> sync, so a one-time ad-hoc sync might be required even if the scheduling is off. We probably want to clarify this point, but it can always be changed later as needed. I'll raise a question about it.

Reply:
Yeah it's an expectation management thing - do we still want to make it possible to force a sync when sync is disabled? cc @danajuratoni @serenachou